mint.json

```diff
@@ -111,6 +111,34 @@
     {
       "group": "Configurations",
       "pages": [
+        {
+          "icon": "user-lock",
+          "group": "Authentication & Personalization",
+          "pages": [
+            "settings/authentication-personalization/shared-features",
+            "settings/authentication-personalization/authentication-vs-personalization",
```

Comment:
I think we should emphasize Auth over Personalization but rn it feels like "Shared features" emphasizes personalization and then we highlight auth in "Authentication vs Personalization". Would changing the order still make sense?

Reply:
I just updated it to separate Authentication's features and Personalization's features entirely. I want to make sure people know they can use Authentication without using any of Personalization's features if they want. Right now Personalization is first but happy to change the order just to emphasize that fact.

Reply:
yeah i think we should change the order

Reply:
done